### DISCUSSION TITLE
"UF2_VOLUME_NAME" definition of XIAO BLE (XIAO nRF52840)

### DISCUSSION THREAD

**Discussion by kitazaki** [Moddable-OpenSource/moddable#1200] [Q&A]
Thank you for great work on nRF52 SoC with Moddable.
XIAO BLE (nRF52840) works fine for my environment with some changes in Manifest.json as below.

```
% diff manifest.json manifest.json_org 
4c4
< 		"UF2_VOLUME_NAME": "XIAO-SENSE"
---
> 		"UF2_VOLUME_NAME": "MODDABLEnRF"
```

I have XIAO BLE (nRF52840) and XIAO BLE (nRF52840) SENSE, which are both mounted on macOS and recognized as "XIAO-SENSE".
Drive name is depends on a bootloader which runs on a device.
The manual does not contain any prerequisites.

**Comment by mkellner** [comment]
Hello @kitazaki,

Thank you for trying the nRF52 port.

If I read your message correctly, you have success without modifying the default bootloader?

The nRF52 document has instructions on how to install the Moddable version of the bootloader at https://www.moddable.com/documentation/devices/nrf52#install-bootloader

For some devices, that is required. I do not have an unmodified XIAO device so I can not test it there immediately.

**Reply by kitazaki** [reply]
@mkellner 
Thank you for quick reply. 🙏
I understand that the Moddable version of the bootloader is needed for XIAO and other devices also.
Let me try and report after.

**Comment by kitazaki** [comment]
The Moddable version of the bootloader works fine with my Seeed XIAO BLE.

I use the ST-Link V2 and XIAO expansion board with XIAO BLE to write the bootloader.
https://www.seeedstudio.com/Seeeduino-XIAO-Expansion-board-p-4746.html

![IMG_2230](https://github.com/Moddable-OpenSource/moddable/assets/23477407/2b23e723-8ded-4e8c-bb85-3c673ec7d3c3)
